src/grpcd/runtime.rs

```diff
@@ -1393,14 +1382,17 @@ impl Farcaster for FarcasterService {
             .map_err(|_| Status::invalid_argument("arbitrating address"))?;
         let monero_address = monero::Address::from_str(&str_monero_address)
             .map_err(|_| Status::invalid_argument("accordant_address"))?;
-        let public_offer = PublicOffer::from_str(&str_public_offer)
-            .map_err(|_| Status::invalid_argument("public offer"))?;
+        let deal =
+            Deal::from_str(&str_deal).map_err(|_| Status::invalid_argument("public deal"))?;
```

Comment:
Global search for `s/public deal/deal/`

Reply:
dc5fe2b699596883ce4d3b775656e20f604299ca